voicevox_engine/app/routers/tts_pipeline.py

```diff
@@ -183,6 +187,7 @@ def accent_phrases(
         text: str,
         style_id: Annotated[StyleId, Query(alias="speaker")],
         is_kana: bool = False,
+        enable_e2k: bool = False,
```

Comment:
ちょっとデフォルト値Trueにしつつ、検討TODO足すのお願いしてもよいでしょうか 🙇 
```suggestion
        enable_e2k: bool = True, # TODO: デフォルト値をどうするか考える
```

こっちのがテストするのに便利そうなのと（外部のコメント読み上げツール使って試してみたい）、デフォルト値どうするかは品質によりそうだなーと思ったので･･･！